### PR TITLE
ocp-client-vm :  default variables for maven and non_root_user

### PR DESCRIPTION
**Pull request by jbride**
A few changes to ocp-client-vm based on running it in the following environments:

1)  RHEL7 vm managed in local libvirtd environment
2)  RHEL7 vm managed in remote Azure environment

### CODE REVIEW

File: ansible/roles/ocp-client-vm/defaults/main.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+
+non_root_os_user: "ec2-user"
```
Comment (fridim) [inline ansible/roles/ocp-client-vm/defaults/main.yml]: there is another variable `remote_user` and `ansible_ssh_user` don't you want to use that instead ?

File: ansible/roles/ocp-client-vm/defaults/main.yml [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+maven_version: "3.5.4"
+docker_device: /dev/xvdb
```
Comment (wkulhanek) [inline ansible/roles/ocp-client-vm/defaults/main.yml]: @jbride  Where is this docker_device being used? I don't see this anywhere in your changes.

File: ansible/roles/ocp-client-vm/tasks/packages.yml [hunk 3]
```diff
@@ -16,36 +16,49 @@
...
-    url: https://www-us.apache.org/dist/maven/maven-3/3.5.4/binaries/apache-maven-3.5.4-bin.tar.gz
+    url: https://www-us.apache.org/dist/maven/maven-3/{{maven_version}}/binaries/apache-maven-{{maven_version}}-bin.tar.gz
     dest: /root/apache-maven.tar.gz
   register: r_geturl
   retries: 5
   delay: 20
   until: r_geturl is succeeded
   tags:
   - install_openshift_client_vm_packages
+  when: 'existing_mvn_version.stat.exists == False'
```
Comment (wkulhanek) [inline ansible/roles/ocp-client-vm/tasks/packages.yml]: @jbride  I'd prefer a block around the "Maven Install". That way the when condition is only evaluated once. See the section "Install everything for student user" as an example.

File: ansible/roles/ocp-client-vm/defaults/main.yml (hunk 1 above)
Comment (wkulhanek) [reply ansible/roles/ocp-client-vm/defaults/main.yml]: @jbride @fridim Agreed. Any reason we couldn't use `remote_user`

**Comment by fridim** [comment]
@jbride any update on this PR ?

**Pull request closed without merging**